db/internal_stats.cc

```diff
@@ -79,9 +79,7 @@ const std::map<InternalStats::InternalDBStatsType, DBStatInfo>
         {InternalStats::kIntStatsWriteStallMicros,
          DBStatInfo{"db.user_write_stall_micros"}},
         {InternalStats::kIntStatsWriteBufferManagerLimitStopsCounts,
-         DBStatInfo{WriteStallStatsMapKeys::CauseConditionCount(
-             WriteStallCause::kWriteBufferManagerLimit,
-             WriteStallCondition::kStopped)}},
+         DBStatInfo{"db.write_buffer_manager_limit_stops"}},
```

Comment:
[Note] Just a name when dumping db stats so does not necessarily to be the same as `WriteStallStatsMapKeys::CauseConditionCount`